pages/docs/apps/cloud.mdx

```diff
@@ -114,7 +114,38 @@ If your app location changes, enable the "Override" switch and edit the URL befo
 
 ## Archive an app
 
-Apps cannot be directly archived. Instead, an app is considered archived when all of its functions are archived.
+Apps can be archived and unarchived at any time. Once an app is archived, all of its functions are archived.
+
+When the app is archived:
+- New function runs will not be triggered.
+- Existing function runs will continue until completion.
+- Functions will be marked as archived but still be visible, including their run history.
+
+
```

Comment:
**suggestion**: Since this does mention that runs continue to completion, it might be a good place to include a link to how to cancel runs because it's related to this concept of what a developer might be seeking.
```suggestion
If you need to cancel all runs prior to completion, read our [cancellation guide](/docs/guides/cancel-running-functions).
```